### DISCUSSION TITLE
issue: handling nested input value bug

### DISCUSSION THREAD

**Discussion by tamirhal** [react-hook-form/react-hook-form#11175] [Q&A]
### Version Number

7.45.3

### Codesandbox/Expo snack

https://codesandbox.io/s/frosty-cannon-fw3qph?file=/src/Input.js

### Steps to reproduce

Hi everyone,
I've noticed a different behavior after upgrading from 4.45.2 to 4.45.3 (also tried 4.47.0)
after the upgrade, the values of nested inputs are not changed

Codesandbox with the unexpected behavior, version 4.45.3 https://codesandbox.io/s/frosty-cannon-fw3qph?file=/src/Input.js

### Expected behaviour

When using setValue for nested input it will change the value as in the link below  

Version 4.45.2 https://codesandbox.io/s/magical-beaver-fp3q72

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
Doesn't support such usage: `resetField("willBecomeNestedInput", { defaultValue: nestedDefaultValues });`

`resetField` should target on the field name, it doesn't support path name.

please refer to the release log: https://github.com/react-hook-form/react-hook-form/releases/tag/v7.45.3
